### DISCUSSION TITLE
blog/tortoise-fine-tuned/

### DISCUSSION THREAD

**Discussion by giscus** [152334H/152334H.github.io#6] [Announcements]
# blog/tortoise-fine-tuned/

Five days ago, I published a blog post, describing why TorToiSe could not be fine-tuned.
Today, I have released a fork of DL-Art-School with TorToiSe fine-tuning code. How did that happen?

https://152334h.github.io/blog/tortoise-fine-tuned/

**Comment by zolero** [comment]
Lets connect! Invited you on Linkedin.
